ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -80,18 +80,15 @@ public void close() {
   void fetchAndRunCommands() {
     final List<QueuedCommand> commands = commandStore.getNewCommands();
     log.trace("Found {} new writes to command topic", commands.size());
-    commands.stream()
-        .filter(c -> c.getCommand().isPresent())
-        .forEach(c -> executeStatement(c.getCommand().get(), c.getCommandId(), c.getStatus()));
+    commands.forEach(c -> executeStatement(c.getCommand(), c.getCommandId(), c.getStatus()));
   }
 
   /**
    * Read and execute all commands on the command topic, starting at the earliest offset.
    * @throws Exception TODO: Refine this.
    */
   public void processPriorCommands() {
-    final RestoreCommands restoreCommands = commandStore.getRestoreCommands();
-    statementExecutor.handleRestoration(restoreCommands);
+    statementExecutor.handleRestoration(commandStore.getRestoreCommands());
```

Comment:
I agree in principal, but left it this way for 5.1.x. Master is already written this way, so we can just switch to this pattern as part of the merge.

> Also, rather than tracking the queries that are created during the restore, can we not just loop over all the statements and then start all persistent queries the engine has at the end?

I had it this way at first, but then switched it to collect the queries in a list because I wanted to do all the `start` calls in the order the statements appear in the log. I don't really have a strong reason for this though. I'm fine with just doing it the way you've suggested.